pkg/agent/openflow/pipeline_windows.go

```diff
@@ -140,3 +140,113 @@ func (c *client) l3FwdFlowToRemoteViaRouting(localGatewayMAC net.HardwareAddr, r
 	}
 	return []binding.Flow{c.l3FwdFlowToRemoteViaGW(localGatewayMAC, *peerPodCIDR, category)}
 }
+
+func (c *client) connectionTrackFlowsAntreaProxyClusterIPSvcHost(category cookie.Category) []binding.Flow {
+	connectionTrackCommitTable := c.pipeline[conntrackCommitTable]
+	// These 2 tables are for traffic SNAT of ClusterIP Service from Windows host.
+	serviceConnectionTrackTable := c.pipeline[serviceConntrackTable]
+	serviceConnectionTrackCommitTable := c.pipeline[serviceConntrackCommitTable]
+
+	return []binding.Flow{
+		// This flow helps reply packet go to serviceConntrackTable to do d-SNAT.
```

Comment:
Suggest not to use "helps". Just say "This flow does something". For example "This flow sends Service reply packets to serviceConntrackTable to do un-SNAT".

Please consider changing all comments.

Reply:
Updated all.